apps/desktop/src-tauri/src/db/migrations/018_tones.sql

```diff
@@ -0,0 +1,17 @@
+-- Create tones table for storing tone definitions and prompt templates
+CREATE TABLE IF NOT EXISTS tones (
+  id TEXT PRIMARY KEY,
+  name TEXT NOT NULL,
+  prompt_template TEXT NOT NULL,
+  is_system INTEGER NOT NULL DEFAULT 0,
+  created_at INTEGER NOT NULL,
+  sort_order INTEGER NOT NULL DEFAULT 0
+);
```

Comment:
We should store which language this tone preference is for. If your current language is == the tone's language, we use it. Else fall back to a built-in one

Reply:
Great point, had a similar thought up here https://github.com/josiahsrc/voquill/pull/8#discussion_r2532059482

The "output in this language" doesn't work at transcription time, but I bet the AI post processing can handle that just fine